code/game/turfs/walls/resin.dm

```diff
@@ -205,4 +205,41 @@
 
 /* Hivelord walls, they start off stronger */
 /turf/closed/wall/resin/regenerating/thick
-	max_integrity = 125
+	max_integrity = 250
+
```

Comment:
???
Undocumented change?

Reply:
Oops as fuck, when I used merge conflict I forgot about this. With the resin armor changes I had to reduce integ to 125, 250 was the original and it isn't anymore.